stix_shifter_modules/msatp/stix_translation/json/to_stix_map.json

```diff
@@ -1,6 +1,269 @@
 {
+  "DeviceAlertEvents": {
```

Comment:
all these changes need to go in stix2.1 mappings as well- `/msatp/stix_translation/json/stix_2_1/to_stix_map.json`